lists/productSkuList.ts

```diff
@@ -0,0 +1,21 @@
+import { relationship, text, timestamp } from '@keystone-next/fields';
+import { list } from '@keystone-next/keystone/schema';
+
+const productSkuList = list({
+    ui: {
+        listView: {
+            initialColumns: ['sku', 'products', 'sku_attribute'],
+        },
+    },
+    fields: {
+        created_at: timestamp({ defaultValue: Date() }),
+        updated_at: timestamp(),
+        deleted_at: timestamp({ isRequired: false }), // the deleted_at and is_deleted fiels are used for soft deletes  
+        is_deleted: text({ isRequired: true, defaultValue: 'false' }),
+        products: relationship({ ref: 'Product.product_sku' }),
```

Comment:
Maybe I don't understand how schemas work in keystone anymore, but shouldn't a productSku have a product_id?